data/hai/unfettered 2 war preparations.txt

```diff
@@ -356,15 +356,15 @@ mission "<uhai> 12: Outfitter Upgrade"
 		log `Helped the Unfettered upgrade their shipyards and received a prototype ship as thanks.`
 		event "unfettered: darkcloak outfitter upgrade" 30
 		conversation
-			`After unloading all the cargo, you go check for He-Yi in the shipyard, but instead you find an Herald, ready to rely a message to you, "Ah, <first>. You have completed your part of the bargain, now you may have your reward.`
-			`	"Go to Firelode, Mera has a very peculiar gift for you, should you prove yourself to be worthy of it.`
+			`After unloading all the cargo, you go check for He-Yi in the shipyard, but instead you find a herald, ready to rely a message to you, "Ah, <first>. You have completed your part of the bargain, now you may have your reward."`
+			`	"Go to Firelode, Mera has a very peculiar gift for you, should you prove yourself to be worthy of it."`
```

Comment:
Not including these " at the end is intentional when it is a multi paragraph quote, like it happens in HR (to be honest I'm not sure of how it works) myself